BREAKING.md

```diff
@@ -84,6 +85,11 @@ iOS:
 |`$range-ios-knob-box-shadow`|`0 3px 1px rgba(0, 0, 0, .1), 0 4px 8px rgba(0, 0, 0, .13), 0 0 0 1px rgba(0, 0, 0, .02)`|`0px 0.5px 4px rgba(0, 0, 0, 0.12), 0px 6px 13px rgba(0, 0, 0, 0.12)`|
 |`$range-ios-knob-width`|`28px`|`26px`|
 
+<h4 id="version-7x-slides">Segment</h4>
```

Comment:
the `id` should be `version-7x-segment` instead of `version-7x-slides` right?

Reply:
Correct - great catch!